<launch>
    <node pkg="espeleo_planner" type="mesh_planner_exploration_node.py" name="mesh_planner_node" output="screen">
        <param name="frame_id" type="string" value="os1_init" />
    </node>
</launch>
